platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -119,4 +131,34 @@ public void init() throws Exception {
 
         return export(Paths.get(exportDirectory));
     }
+
+    BundleContext getBundleContext() {
+        return FrameworkUtil.getBundle(ConfigurationMigrationManager.class)
+                .getBundleContext();
+    }
+
+    private Collection<MigrationWarning> exportMigratables(Path exportDirectory) {
+        List<MigrationWarning> warnings = new LinkedList<>();
+
+        for (ServiceReference<Migratable> serviceReference : getMigratableServiceReferences()) {
+
+            Migratable migratable = getBundleContext().getService(serviceReference);
+            MigrationMetadata migrationMetadata =
+                    migratable.export(exportDirectory.resolve((String) serviceReference.getProperty(
+                            Constants.SERVICE_PID)));
+            warnings.addAll(migrationMetadata.getMigrationWarnings());
+        }
+
+        return warnings;
+    }
+
+    private Collection<ServiceReference<Migratable>> getMigratableServiceReferences() {
+        try {
+            return getBundleContext().getServiceReferences(Migratable.class, null);
+        } catch (InvalidSyntaxException e) {
+            String message = "Failure to export, internal error occurred.";
```

Comment:
Changed log message. Exception and stack trace will also get logged which should help too.